compact/display_geoviewer.xml

```diff
@@ -82,6 +82,9 @@
     <vis name="HcalBarrelStaveVis"    ref="AnlBlue"   showDaughters="false" visible="true"/>
     <vis name="HcalBarrelLayerVis"    ref="AnlBlue"   showDaughters="true"  visible="true"/>
     <vis name="HcalEndcapVis"         ref="AnlBlue"   showDaughters="false" visible="true"/>
+    <vis name="LFHCALVis"             ref="AnlRed"   showDaughters="true" visible="true"/>
+    <vis name="LFHCALLayerSteelVis"   ref="AnlBlue"   showDaughters="false" visible="true"/>
+    <vis name="LFHCALLayerTungstenVis" ref="AnlDarkBlue"   showDaughters="false" visible="true"/>
```

Comment:
I'm surprised this is shorter. Doesn't this lead to missing visualization attribute definitions when running with this set of display definitions? In any case, unless you are using the difference in visualization attributes between geoviewer vs regular, this should go into the detector's xml file.